src/qmxgraph/page/api.js

```diff
@@ -1277,96 +1338,100 @@ graphs.Api.prototype.registerTerminalChangedHandler = function registerTerminalC
  * boolean indicating if the changed terminal is the source (or target), id of the new terminal,
  * port id used in the new terminal, id of the old terminal, port id used in the old terminal.
  */
-graphs.Api.prototype.registerTerminalWithPortChangedHandler = function registerTerminalWithPortChangedHandler (handler) {
-    "use strict";
+graphs.Api.prototype.registerTerminalWithPortChangedHandler =
```

Comment:
=/
https://prettier.io/docs/en/ignore.html#javascript

> A JavaScript comment of // prettier-ignore will exclude the next node in the abstract syntax tree from formatting.

I ask myself if by "node" they mean just a node or the full sub tree.